### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by ntqdinh-axonivy**

### CODE REVIEW

File: adobe-esign-connector-product/product.json [hunk 1]
```diff
@@ -23,12 +23,12 @@
...
-        "dependencies": [
+        "projects": [
           {
-            "groupId": "MY-GROUP-ID",
-            "artifactId": "MY-PRODUCT-NAME",
+            "groupId": "com.axonivy.connector.adobe.esign",
+            "artifactId": "adobe-esign-connector",
             "version": "${version}",
             "type": "iar"
           }
```
Comment (ivy-rew) [inline adobe-esign-connector-product/product.json:34]: the maven-import installer is here, to add projects to the workspace, without adding any dependencies to existing projects, so this is naturally the place where add "demo" artifacts ... which the user freely remove later on.
While the maven-dependency installer is used to add dependencies to the project, e.g to add the connector itself.

However, here we forgot the demo, but added the connector twice @ntqdinh-axonivy .
Please review these files manually in the future. Do so by installing local artifacts of this workspace using `mvn clean install` and consume the product.json with  the CTRL+Alt+I shortcut in the Designer. 

https://github.com/axonivy-market/adobe-acrobat-sign-connector/pull/62

Comment (ntqdinh-axonivy) [reply adobe-esign-connector-product/product.json:34]: Hi @ivy-rew  
Thanks for your validation. It was the first time I onboarded a new product to the marketplace. So there were some silly mistakes when not fully understand the differences between maven import/ dependency at that time. We did have some internal meetings for these mistakes and I think it has been improved day by day.

Comment (ivy-rew) [reply adobe-esign-connector-product/product.json:34]: I'm glad to see your progress here. Thanks for your efforts :handshake:

Comment (ivy-lmu) [reply adobe-esign-connector-product/product.json:34]: Hi @ntqdinh-axonivy,
please note that the following installer configuration has similar problems https://github.com/axonivy-market/ai-assistant/blob/master/ai-assistant-product/product.json

**Pull request merged**